.github/workflows/publish-prisma-schema-wasm.yml

```diff
@@ -30,26 +31,19 @@ jobs:
           ref: ${{ github.event.inputs.enginesHash }}
       - uses: cachix/install-nix-action@v23
 
-      #
-      # Build
-      #
-
-      - run: nix build .#prisma-schema-wasm
-
-      #
-      # Publish
-      #
+      - name: Build
+        run: nix build .#prisma-schema-wasm
 
       - uses: actions/setup-node@v3
         with:
-          node-version: '14.x'
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org/'
 
-      - name: Set up NPM token
-        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+      - name: Update version in package.json
+        run: PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
 
-      - run: |
-          PACKAGE_DIR=$( nix run .#renderPrismaSchemaWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
-          npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
+      - name: Publish @prisma/prisma-schema-wasm
+        run: npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
```

Comment:
Does `PACKAGE_DIR=...` set the env variable for the next `run` in the next step as well?

Reply:
Right... Thanks!

Reply:
It's definitely weird, but fixed now